cinema/permissions.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework import permissions
+
+
+class IsAdminOrIfAuthenticatedReadOnly(permissions.BasePermission):
+
+    def has_permission(self, request, view):
+        if (request.method in permissions.SAFE_METHODS
+                and request.user.is_authenticated):
+            return True
+
+        return request.user.is_authenticated and request.user.is_staff

```

Comment:
It is not required to split to conditions here
You can make it as 1 if statement(because each check returns boolean) but that's good as well